lib/discordrb/data/member.rb

```diff
@@ -247,10 +247,17 @@ def server_unmute
     end
 
     # Bans this member from the server.
-    # @param message_days [Integer] How many days worth of messages sent by the member should be deleted.
+    # @param message_days [Integer] How many days worth of messages sent by the member should be deleted. This will be deprecated in 4.0.
```

Comment:
Shouldn’t this parameter be deprecated now and removed in 4.0?

Reply:
As per what @swarley has [said in DAPI](https://discord.com/channels/81384788765712384/381891448884428801/1297344043012980807) I was told to keep the argument as is and add seconds as an optional KWARG.

Reply:
I think that comment referred to keeping it in version `3.x`, not in `4.0`. In semantic versioning (semver), a breaking change, like a backwards incompatible method signature change, is only allowed during major version changes, such as `3.x` to `4.0`. A deprecation is a warning that it will be removed in a future version, in this case likely `4.0`. Therefore, it should be deprecated now and removed in `4.0`, not, as the comment claims, be deprecated in `4.0`, which would imply its existence in `4.0` (and thus its removal not earlier than in `5.0`).

Reply:
> I think that comment referred to keeping it in version `3.x`, not in `4.0`. In semantic versioning (semver), a breaking change, like a backwards incompatible method signature change, is only allowed during major version changes, such as `3.x` to `4.0`. A deprecation is a warning that it will be removed in a future version, in this case likely `4.0`. Therefore, it should be deprecated now and removed in `4.0`, not, as the comment claims, be deprecated in `4.0`, which would imply its existence in `4.0` (and thus its removal not earlier than in `5.0`).

Oh whoops. Seems like I misunderstood what you mean.